2.4/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -86,8 +86,12 @@ function _wait_for_mongo() {
 # To get list of endpoints, you need to have headless Service named 'mongodb'.
 # NOTE: This won't work with standalone Docker container.
 function endpoints() {
-  service_name=${MONGODB_SERVICE_NAME:-mongodb}
-  dig ${service_name} A +search +short 2>/dev/null
+  if [ -n "${MONGODB_REPLICA_MEMBERS+x}" ]; then
```

Comment:
> Why we can't use [ -n "${MONGODB_REPLICA_MEMBERS:-}" ] instead?

@php-coder The if condition have to be true even for set but null value (for MONGODB_REPLICA_MEMBERS=""), therefore I used +.

Something different should be used?